luigi/interface.py

```diff
@@ -132,13 +136,13 @@ def _schedule_and_run(tasks, worker_scheduler_factory=None, override_defaults=No
     :param worker_scheduler_factory:
     :param override_defaults:
     :return: True if all tasks and their dependencies were successfully run (or already completed);
-             False if any error occurred.
+             False if any error occurred. It will return a detailed response of type LuigiRunResult
+             instead of a boolean if detailed_summary=True in override_defaults.
     """
 
     if worker_scheduler_factory is None:
         worker_scheduler_factory = _WorkerSchedulerFactory()
-    if override_defaults is None:
-        override_defaults = {}
+    override_defaults = {} if override_defaults is None else override_defaults
```

Comment:
No need to change, it just makes the diff longer. :)

Reply:
I added this to meet coverage 🙈 This line was causing a build failure 🙈 An explicit kwarg should solve this 😄

Reply:
Can  you just do

    override_defaults = override_defaults or {}

?

Reply:
Wow so elegant! Yes! 🎩